### ISSUE TITLE
Can this model train the Pascal VOC dataset?

### ISSUE THREAD

**Issue by 1451595897** [tadax/m2det#1]
Hello，I want to ask you,can this model train the Pascal VOC datasets?

**Comment by tadax** [comment]
@1451595897 Yes you can. Prepare text files for annotations in the format as (class_index, min_x, min_y, max_x, max_y). Please check [mscoco/process.py](https://github.com/tadax/m2det/blob/master/mscoco/process.py) as a reference.
